test/integration/BaseIntegration.t.sol

```diff
@@ -0,0 +1,301 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+// external
+import { Test } from "forge-std/Test.sol";
+import { MessageHashUtils } from "@openzeppelin/contracts/utils/cryptography/MessageHashUtils.sol";
+import { MetaTx } from "@storyprotocol/core/lib/MetaTx.sol";
+import { IIPAccount } from "@storyprotocol/core/interfaces/IIPAccount.sol";
+import { AccessPermission } from "@storyprotocol/core/lib/AccessPermission.sol";
+import { IAccessController } from "@storyprotocol/core/interfaces/access/IAccessController.sol";
+import { ILicensingModule } from "@storyprotocol/core/interfaces/modules/licensing/ILicensingModule.sol";
+import { ICoreMetadataModule } from "@storyprotocol/core/interfaces/modules/metadata/ICoreMetadataModule.sol";
+import { ICoreMetadataViewModule } from "@storyprotocol/core/interfaces/modules/metadata/ICoreMetadataViewModule.sol";
+
+// contracts
+import { IStoryProtocolGateway as ISPG } from "../../contracts/interfaces/IStoryProtocolGateway.sol";
+
+// script
+import { DeployHelper } from "../../script/utils/DeployHelper.sol";
+
+// test
+import { MockIPGraph } from "../mocks/MockIPGraph.sol";
+import { MockERC20 } from "../mocks/MockERC20.sol";
+import { MockERC721 } from "../mocks/MockERC721.sol";
+import { Users, UserSecretKeys, UsersLib } from "../utils/Users.t.sol";
+
+contract BaseIntegration is Test, DeployHelper {
+    /// @dev RPC URL for the fork
+    string internal constant RPC_URL = "https://testnet.storyrpc.io"; // Iliad Testnet RPC
+
+    uint256 internal constant FORK_CHAIN_ID = 1513; // Iliad Testnet Chain ID
+
+    /// @dev Users struct to abstract away user management when testing
+    Users internal u;
+
+    /// @dev UserSecretKeys struct to abstract away user secret keys when testing
+    UserSecretKeys internal sk;
+
+    address internal minter;
+    address internal feeRecipient;
+
+    /// @dev Minter's secret key
+    uint256 internal minterSk;
+
+    /// @dev Multisig address
+    address internal multiSig = 0xf398C12A45Bc409b6C652E25bb0a3e702492A4ab;
```

Comment:
It would be better not hardcode the EOA wallet in the code, instead using environment variable to pass in.
You might need to setup github action to read env var.